SConstruct

```diff
@@ -192,6 +192,14 @@ target_tool.options(opts)
 opts.Update(env)
 Help(opts.GenerateHelpText(env))
 
+# Process platform argument
+if env.get("platform", default_platform) == "linux":
+    env.Append(CPPDEFINES=["LINUX_ENABLED", "UNIX_ENABLED"])
```

Comment:
Is there a reason that GDExtension doesn't build the "linuxbsd" platform instead of just "linux"?